### ISSUE TITLE
Are all disposers called when root element is disposed?

### ISSUE THREAD

**Issue by hesselmeulenbeld** [amcharts/amcharts5#923]
I dispose of the chart using root.dispose().
But when analyzing memory usage in the browser there are still RoundedRectangles in memory because they are kept in memory by
Sprite.ts -> _tooltipDp
This is with using showTooltipOn == "click" and chrome says it is in the lines:
```
$utils.addEventListener(document, "click", (_ev) => {
    this.hideTooltip();
})
```

Should I dispose of the data in another way or is this a bug?

**Comment by hesselmeulenbeld** [comment]
It looks like the same issue is in XYCursor.ts with the keyboardevents:

```
if ($utils.supports("keyboardevents")) {
	this._disposers.push($utils.addEventListener(document, "keydown", (ev: KeyboardEvent) => {
		this._handleLineMove(ev.keyCode);
	}));
}
```

in _afterNew().

**Comment by martynasma** [comment]
It looks like the first event is not being disposed. We'll fix it in the next version.

That XYChart event is correctly being pushed to disposers, and should be disposed of when disposing root. Can you confirm it's not being disposed, or just observing the code?

**Comment by martynasma** [comment]
Fixed in [5.3.13](https://github.com/amcharts/amcharts5/releases/tag/5.3.13).

## [5.3.13] - 2023-05-30

### Added
- New setting `forceLTR` (default: `false`) on `NumberFormatter`. If set to `true`, will force all formatted numbers to be LTR, regardless of the direction settings on chart/page.
- `legend: true` added to `dataContext` of a legend bullet `DataItem` (bullets are created for LineSeries), just so it is possible to distinguish between bullets created for series and legend.

### Fixed
- `Hierarchy` was not not keeping its node toggle state after resize.
- In some cases `DateRangeSelector` was generating an error when zooming `SotckChart`.
- Always-on / on-click `Tooltip` was not being disposed properly when `Root` element was disposed.
- Memory leak was happening when disposing a `Container` with `verticalScrollbar` enabled.
- `PieSeries` was erroring if its `stroke` setting was set.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
